docs/api_reference/messages_api/post_send_messages.md

```diff
@@ -143,3 +147,109 @@ curl -X POST "https://api.callbell.eu/v1/messages/send" \
     }
   }'
 ```
+
+## Send Template Messages
+
+:::info
+This is only available for accounts using the official **WhatsApp Business API** integration.
+:::
+
+You can use the API to send an approved [Template](/api_reference/object_types/template) Message
+In order to send template messages `template_uuid` and `optin_contact` **must** be present in the payload.
+
+Use the [Templates API](/api_reference/template_messages_api/introduction) to the get the `template_uuid`s
```

Comment:
Maybe this could be moved at the end of the section? In here it's slightly confusing IMO